google/cloud/storage/internal/object_requests.h

```diff
@@ -197,6 +197,15 @@ class DeleteObjectRequest
                                   IfMetagenerationNotMatch, UserProject> {
  public:
   using GenericObjectRequest::GenericObjectRequest;
+
+  std::string const& upload_id() const { return upload_id_; }
```

Comment:
This change seems unrelated, this is a `DeleteObjectRequest`, not a `DeleteResumableUploadRequest`, it seems the proto APIs make no distinction, but we do here.

Reply:
Fixed.